journal/modernized/04-chapter-3.adoc

```diff
@@ -235,6 +240,7 @@ for I felt the Lord`'s power strengthening me.
 
 So he went out of the room;
 and this was but an apology to get an opportunity for the priest and
+//Should apology be changed to excuse in the modernized?
```

Comment:
Yes, I would change apology to excuse here in the modernized version. Good call.